okhttp/src/commonJvmAndroid/kotlin/okhttp3/internal/-MediaTypeCommon.kt

```diff
@@ -45,16 +45,15 @@ private val PARAMETER = Regex(";\\s*(?:$TOKEN=(?:$TOKEN|$QUOTED))?")
  * @throws IllegalArgumentException if this is not a well-formed media type.
  */
 internal fun String.commonToMediaType(): MediaType {
-  val typeSubtype: MatchResult =
-    TYPE_SUBTYPE.matchAtPolyfill(this, 0)
+  val typeSubtype = TYPE_SUBTYPE.matchAt(this, 0)
```

Comment:
Seems like Spotless wants
```suggestion
  val typeSubtype =
    TYPE_SUBTYPE.matchAt(this, 0)
```